test/carryToken.js

```diff
@@ -0,0 +1,96 @@
+// The Carry token and the tokensale contracts
+// Copyright (C) 2018 Carry Protocol
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+const { assertEq, assertFail } = require("./utils");
+
+const CarryToken = artifacts.require("CarryToken");
+
+contract('CarryToken', async (accounts) => {
+    const instance = await CarryToken.deployed();
+    const [sender, receiver] = accounts;
```

Comment:
It could be spread directly in the parameter like this:

```js
contract('CarryToken', async ([sender, receiver]) => {
    const instance = await CarryToken.deployed();
    // ......
```